Dockerfile

```diff
@@ -0,0 +1,83 @@
+FROM harbor.k8s.libraries.psu.edu/library/ruby-2.6.6-node-15:20210507 as base
```

Comment:
Could go ahead and move to ruby 2.7.3? Or is that moving too many things at once?

Reply:
I think we could tackle upgrading ruby as another PR.